core/arch/arm/kernel/cache_helpers_a64.S

```diff
@@ -102,6 +102,7 @@ LOCAL_FUNC do_dcsw_op , :
 	cbz	x3, exit
 	adr	x14, dcsw_loop_table	// compute inner loop address
 	add	x14, x14, x0, lsl #5	// inner loop is 8x32-bit instructions
+	BTI(add	x14, x14, x0, lsl #2)	// inner loop is + "bti j" instruction
```

Comment:
The preferred style is:
``` C
BTI(	add	x14, x14, x0, lsl #2)
```
With `BTI(` to the left with no whitespace in front, followed by a  tab to line up with the other instructions.
compare with how we use the `UNWIND()` macro.
Please fix here and at the other places.

Reply:
Changed